lib/tasks/invite_code.rake

```diff
@@ -1,16 +1,25 @@
 
 
 desc "Generate a batch of invite codes"
-task :generate_invites, [:n] => :environment do |task, args|
+task :generate_invites, [:n, :u] => :environment do |task, args|
 
-    codes = args.n
-    codes = codes.to_i
+  codes = args.n
+  codes = codes.to_i
 
-    codes.times do |x|
+  if args.u == nil
+    max_uses = 1
+
+  elsif
+    max_uses = args.u
+    max_uses = max_uses.to_i
```

Comment:
We actually didn't need it, I just stole some code from a few lines above there. :) 
Cleaned up now!